src/lib/AnnotationControls.ts

```diff
@@ -146,28 +144,29 @@ export default class AnnotationControls {
      * Region comment button click handler
      */
     private handleClick = (onClick: ClickHandler, mode: AnnotationMode) => (event: MouseEvent): void => {
-        const prevActiveControl = this.currentActiveControl;
+        const prevActiveControl = this.currentMode;
 
         this.resetControls();
 
         if (prevActiveControl !== mode) {
-            this.currentActiveControl = mode as AnnotationMode;
+            this.currentMode = mode as AnnotationMode;
             this.controlsMap[mode]();
         }
 
-        onClick({ activeControl: this.currentActiveControl, event });
+        onClick({ activeControl: this.currentMode, event });
```

Comment:
Is `activeControl` used anywhere? If not, can it be removed? If so, can it be renamed?